packages/disputer/src/disputer.js

```diff
@@ -199,13 +199,10 @@ class Disputer {
         gasPrice: this.gasEstimator.getCurrentFastPrice()
       };
 
-      const inputPrice = disputeableLiquidation.price;
-
       this.logger.debug({
         at: "Disputer",
```

Comment:
`inputPrice` is no longer necessary to add to the logs because it is included in the `liquidation` object